.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,17 @@
+  # - name: Discord Chat
+  #   url: https://chat.vuejs.org
+  #   about: Ask questions and discuss with other Vue users in real time.
+  # - name: Questions & Discussions
+  #   url: https://github.com/vuejs/core/discussions
+  #   about: Use GitHub discussions for message-board style questions and discussions.
+  # - name: Patreon
+  #   url: https://www.patreon.com/evanyou
+  #   about: Love Vue.js? Please consider supporting us via Patreon.
+  # - name: Open Collective
+  #   url: https://opencollective.com/vuejs/donate
+  #   about: Love Vue.js? Please consider supporting us via Open Collective.
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)